### PR TITLE
Small fixes on attachments

### PR DESCRIPTION
**Pull request by jespino**

### CODE REVIEW

File: taiga/models/models.py [hunk 1]
```diff
@@ -218,8 +221,13 @@ def create(self, project, object_id, attached_file, **attrs):
         :param attrs: optional attributes for the :class:`Attachment`
         """
         attrs.update({'project': project, 'object_id': object_id})
+
+        attachment = attached_file
+        if isinstance(attached_file, str):
```
Comment (astagi) [inline taiga/models/models.py]: @jespino thanks for contributing! Can you please add a test for this?

**Comment by jespino** [comment]
Sure!, now you have the tests for this, If you prefer it, I can make it more robust checking if the param is a str, and then open the path, or if is something that inherit from IOBase, and otherwise raise an exception.

**Comment by astagi** [comment]
Thanks @jespino, looks great! yeah a more robust checking would be awesome! waiting for that then! :smile:

**Comment by astagi** [comment]
Any updates here @jespino ?

**Comment by jespino** [comment]
Now ready! Sorry for the delay

**Comment by astagi** [comment]
No problem @jespino , looks awesome! thank you :smile:

**Pull request merged**